Packs/CTIX/Integrations/CTIXv3/CTIXv3.py

```diff
@@ -1557,20 +1641,253 @@ def file(client: Client, args: Dict[str, Any]) -> List[CommandResults]:
     return get_lookup_threat_data_command(client, args)
 
 
+def get_all_notes(client: Client, args: Dict[str, Any]) -> List[CommandResults]:
+    page = args["page"]
+    page = check_for_empty_variable(page, 1)
+    page_size = args["page_size"]
+    page_size = check_for_empty_variable(page_size, 10)
+    response = {}
+    object_id = args.get("object_id", "")
+    params = {
+        "page": page,
+        "page_size": page_size,
+    }
+
+    if object_id:
+        if not is_valid_uuid(object_id):
+            return_error("Error: The object ID that was provided is not valid.")
+        else:
+            params['object_id'] = object_id
+
+    client_url = client.base_url + "ingestion/notes/"
+    response = client.get_http_request(client_url, **params)
+    notes_list = response.get("data", {}).get("results", [])
+    notes_list = no_result_found(notes_list)
+
+    if isinstance(notes_list, CommandResults):
+        return [notes_list]
+    else:
+        results = []
+        for note in notes_list:
+            results.append(
+                CommandResults(
+                    readable_output=tableToMarkdown("Note Data", note, removeNull=True),
+                    outputs_prefix="CTIX.Note",
+                    outputs_key_field="id",
+                    outputs=note,
+                )
+            )
+        return results
+
+
+def get_note_details(client: Client, args: Dict[str, Any]) -> CommandResults:
+    id = args["id"]
+    client_url = client.base_url + f"ingestion/notes/{id}/"
+    response = client.get_http_request(client_url)
+
+    if response['status'] == HTTPStatus.BAD_REQUEST:
+        return CommandResults(
```

Comment:
maybe its better to return error in this case

Reply:
Fixed.